### DISCUSSION TITLE
Session sync status indicator in the UI

### DISCUSSION THREAD

**Discussion by tmdarlan** [f1multiviewer/issue-tracker#112] [Feature requests] [answered]
Some sort of indication showing if the session has been synced.

**Comment by JJWatMyself** [comment]
This is an amazing idea and eliminates the need to check what is or not synced.

**Comment by jeppeolesen** [comment, marked as answer]
This has been implemented and released in MultiViewer 2.0, which is now available in the beta release channel.
<img width="1187" alt="Screenshot 2025-06-29 at 15 34 10" src="https://github.com/user-attachments/assets/83cdca81-e1d4-4ed3-aef7-fd2a538d2887" />
